en/docs/guides/tenants/customize-automated-mails.md

```diff
@@ -116,7 +116,7 @@ Once you have set the template language for the user, it is also necessary to sp
     !!! tip
     
         For example, use **fr\_fr** for the French locale, **pt\_br** for Brazil locale and **it\_it** for the Italian locale. This is the format used for country locales in the Identity Server.
-        For more information on country locales, see [Locale codes](http://www.science.co.il/language/Locale-codes.php).
+        For more information on country locales, see [Locale codes](https://www.ibm.com/docs/en/radfws/9.7?topic=overview-locales-code-pages-supported).
```

Comment:
```suggestion
        For more information on country locales, see [Locale codes]({{base_path}}/references/supported-locales/).
```

Reply:
We have added a doc with the supported locales to our doc space using PR https://github.com/wso2/docs-is/pull/3943/files

Therefore lets link that doc here.